rasa/core/test.py

```diff
@@ -351,6 +359,37 @@ def _clean_entity_results(
     return cleaned_entities
 
 
+def _intent_response_key_from_parsed_data(parsed: Dict[Text, Any]) -> Text:
```

Comment:
Could we add a docstring here? This method is doing some black magic, would be great to have a guide for the future 👍